ui-modules/home/app/views/about/about.controller.js

```diff
@@ -67,4 +72,139 @@ export function aboutStateController($scope, brBrandInfo, version, states) {
         buildCommitId: BUILD_COMMIT_ID,
         brooklynVersion: BROOKLYN_VERSION,
     };
+
+    this.container = $element[0];
+    this.now = Date.now();
+    this.expectedNodeCounter = Object.keys(this.states.nodes).length;
+    this.template = 'haStatusTemplate';
+
+    let modalInstance = null;
+
+    this.openDialog = function (states, nodeId, serverApi, container) {
+        if (!modalInstance) {
+            modalInstance = $uibModal.open({
+                template: nodeManagementTemplate,
+                controller: ['$scope', '$uibModalInstance', 'node', 'serverApi', nodeManagementController],
+                controllerAs: 'vm',
+                backdrop: 'static',
+                windowClass: 'quick-launch-modal',
+                size: 'md',
+                resolve: {
+                    node: function () {
+                        return states.nodes[nodeId];
+                    },
+                    serverApi: function () {
+                        return serverApi;
+                    }
+                }
+            });
+            modalInstance.result.then(
+                (promiseList) => {
+                    this.template = 'spinnerTemplate';
+                    Promise.allSettled(promiseList).then((values) => {
+                        let event = new CustomEvent('update-states', {});
+                        container.dispatchEvent(event);
+                    });
+                    modalInstance = null;
+                },
+                () => {
+                    this.template = 'spinnerTemplate';
+                    let event = new CustomEvent('update-states', {});
+                    container.dispatchEvent(event);
+                    modalInstance = null;
+                }
+            );
+        }
+
+    };
+
+    function nodeManagementController($scope, $uibModalInstance, node, serverApi) {
+
+        this.node = node;
```

Comment:
I suggest to create reference variable for `this` to reference the instance of the controller.
```
let mgmtController = this;
mgmtController.node = node;
// etc.
```

So that it is clear what instance does `this` reference to:
```
mgmtController.setHaStatus = function () {
    let result = serverApi.setHaStatus(this.newStatus);
    this.node.status = this.newStatus; // this is likely to be a reference to function() instance, not the controller.
    // or ?
    mgmtController.setHaStatus.node.status; // explicit reference to the controller instance.
}
```

Reply:
Applies to all functions defined below.

Reply:
Controller instance could be named asa `vm`, the same as `controllerAs: 'vm',`, instead of `mgmtController`.
See examples in blueprint-composer, e.g. spec-editor.